pkg/scheduler/factory/plugins.go

```diff
@@ -167,6 +167,17 @@ func InsertPredicateKeyToAlgorithmProviderMap(key string) {
 	return
 }
 
+// InsertPriorityKeyToAlgorithmProviderMap inserts a priority function to all algorithmProviders which are in algorithmProviderMap.
+func InsertPriorityKeyToAlgorithmProviderMap(key string) {
+	schedulerFactoryMutex.Lock()
+	defer schedulerFactoryMutex.Unlock()
+
+	for _, provider := range algorithmProviderMap {
+		provider.PriorityFunctionKeys.Insert(key)
+	}
+	return
```

Comment:
Looks like no need to `return`

Reply:
Just a coding style to indicate end of function, I don't have a preference though.